app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class PasswordsController < Devise::PasswordsController
+      skip_before_action :verify_authenticity_token
+
+      protected
+
+      def respond_with(*resource)
+        user = resource.first
+
+        if user.blank?
+          render json: {}, status: :created
+        else
+          render json: { error: user.errors.full_messages.to_sentence }, status: :not_found
```

Comment:
Returning `not_found` actually exposes us to user enumeration as an attacker can know what email exists. So it's usually better to always return success with a message like "If an account matched the email provided, a reset email has been sent". I think that's the default behaviour of Devise.